cmd/soroban-rpc/internal/methods/health.go

```diff
@@ -8,24 +8,36 @@ import (
 	"github.com/creachadair/jrpc2"
 	"github.com/creachadair/jrpc2/handler"
 
+	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/events"
 	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/transactions"
 )
 
 type HealthCheckResult struct {
-	Status string `json:"status"`
+	Status       string `json:"status"`
+	LatestLedger uint32 `json:"latestLedger"`
+	FirstLedger  uint32 `json:"firstLedger"`
```

Comment:
Can we change it to `oldestLedger` to keep it consistent with `getTransaction` where we already return this value

Reply:
sure